docs/data/routes/docs/client-frameworks/nextjs/nextjs-static-export/en.md

```diff
@@ -18,24 +18,10 @@ These instructions you should apply in order to run `next export`:
 	1. Delete i18n configuration in `next.config.js`.
 	1. Define language in `package.json` using `config.language` which will be used during export.
 1. Remove `rewrites` in `next.config.js`.
-1. Define `getStaticPaths` in `[[...path]].tsx` and use:
-	```js
-	import { GetStaticPaths } from 'next';
-	import { graphQLSitemapService } from 'lib/graphql-sitemap-service';
-	import config from 'temp/config';
-	import { config as packageConfig } from '../../package.json';
-
-	export const getStaticPaths: GetStaticPaths = async (context) => {
-	  const ROOT_ITEM = `/sitecore/content/${config.jssAppName}/home`;
-	  const paths = await graphQLSitemapService.fetchExportSitemap(packageConfig.language, ROOT_ITEM);
-
-	  return {
-    	paths,
-    	fallback: false,
-	  };
-	}
-	```
-	> The `fallback: true` and `fallback: 'blocking'` modes of `getStaticPaths` are not supported when using next export.
 1. Remove usage of `<VisitorIdentification />` component in `src/Layout.tsx`, since Visitor Identification not available.
 1. Define `PUBLIC_URL` in `.env`.
-1. Add script `"export": "npm-run-all --serial bootstrap next:build next:export"` in package.json.
+1. Add scripts in `package.json`:
+	* `"next:export": "next export"`.
+	* For disconnected mode: `"export": "cross-env-shell JSS_MODE=disconnected PORT=3042 EXPORT_MODE=true \"npm-run-all --serial bootstrap next:build next:export\""`,
```

Comment:
@ambrauer Look at here: https://github.com/Sitecore/jss/blob/dev/samples/nextjs/scripts/bootstrap.ts#L18-L19
So if we are in disconnected mode, our `sitecoreApiHost` - will be disconnected server, so we need to pass correct PORT, default number is 3042

Reply:
@illiakovalenko Got it, didn't realize we had that in bootstrap. I guess, how is this different than the `PROXY_PORT` we use elsewhere for disconnected mode (next.config.js and in [proxy script](https://github.com/Sitecore/jss/blob/dev/samples/nextjs/scripts/disconnected-mode-proxy.ts#L26))?

Reply:
@ambrauer You should run disconnected server when you want to run export, because you should have ability to execute requests to `layout-service`, `dictionary-service`. In this case I will use PROXY_PORT everywhere, instead of PORT